docs/rp2/quickref.rst

```diff
@@ -112,7 +121,7 @@ Use the ``machine.PWM`` class::
 ADC (analog to digital conversion)
 ----------------------------------
 
-How does the ADC module work?
+RP2040 has five ADC channels in total, four of which are 12-bit SAR based ADCs: GP26, GP27, GP28 and GP29. The input signal for ADC0, ADC1 and ADC2 can be connected with GP26,GP27,GP28. The fourth is used to measure the VSYS voltage on the board.The standard ADC range is 0-3.3V.
```

Comment:
This is good information but we should clarify how the fifth ADC channel can be used.